src/components/views/rooms/RoomHeader.js

```diff
@@ -110,7 +110,7 @@ module.exports = React.createClass({
                 var searchStatus;
                 // don't display the search count until the search completes and
                 // gives us a non-null searchCount.
-                if (this.props.searchInfo && this.props.searchInfo.searchCount !== null) {
+                if (this.props.searchInfo && this.props.searchInfo.searchCount != null) {
```

Comment:
> We want to distinguish between 0 ('there are no search results') and undefined ('we don't yet know if there are any search results').

Right, I see now. I really dislike the use of `!= null` checks when the thing being checked is an integer which can be `0` because of Javascript's lovely coercion rules. Whilst this may work in this case, I'd rather we didn't assume that the reader is aware of all the coercion gotchas. Can we replace this with `&& (this.props.searchInfo.searchCount !== undefined && this.props.searchInfo.searchCount !== null)` please? It's longer to type, but less error-prone when reading I think.